Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Asset/AssetManager.h>
+#include <AzCore/Console/IConsole.h>
+
+#include <AzFramework/API/ApplicationAPI.h>
+
+#include <EditorColliderComponent.h>
+#include <EditorShapeColliderComponent.h>
+#include <EditorRigidBodyComponent.h>
+#include <EditorStaticRigidBodyComponent.h>
+
+#include <Editor/Source/Components/Conversion/PrefabConversionUtils.h>
+
+namespace PhysX
+{
+    void FixPrefabsWithColliderComponents(const AZ::ConsoleCommandContainer& commandArgs);
+
+    AZ_CONSOLEFREEFUNC(
+        "ed_physxFixPrefabsWithColliderComponents",
+        FixPrefabsWithColliderComponents,
+        AZ::ConsoleFunctorFlags::Null,
+        "Finds entities with collider components and no rigid bodies and fixes them by adding a static rigid body component.");
```

Comment:
Instead of saying it fixes them, maybe you could say something like "updates them to the new pattern which requires a static rigid body component"?

Reply:
Personally I would go for "update" rather than "fix" in the name of the function as well.

Reply:
I can change that.